selfdrive/hardware/tici/hardware.h

```diff
@@ -11,14 +11,25 @@ class HardwareTici : public HardwareNone {
 public:
   static constexpr float MAX_VOLUME = 1.0;
   static constexpr float MIN_VOLUME = 0.4;
+  static constexpr float SECONDS_IN_HOUR = 60*60;
+  static constexpr float HOURLY_BRIGHTNESS_DECREASE = 5;
+  static constexpr float BRIGHTNESS_LIMIT_MIN = 30;
+  static constexpr float BRIGHTNESS_LIMIT_MAX = 100;
+  static constexpr float MAX_BRIGHTNESS_HOURS = 4;
+
   static bool TICI() { return true; }
   static std::string get_os_version() {
     return "AGNOS " + util::read_file("/VERSION");
   };
 
   static void reboot() { std::system("sudo reboot"); };
   static void poweroff() { std::system("sudo poweroff"); };
-  static void set_brightness(int percent) {
+  static void set_brightness(int percent, float ui_running_time) {
```

Comment:
I would just put this all in `Device::updateBrightness` until it gets moved to agnos

Reply:
done